doc/api/index.md

```diff
@@ -63,6 +63,8 @@
     - [getPreferredAudioTracks](#meth-getPreferredAudioTracks)
     - [setPreferredTextTracks](#meth-setPreferredTextTracks)
     - [getPreferredTextTracks](#meth-getPreferredTextTracks)
+    - [disableVideoTrack](#meth-disableVideoTrack)
+    - [disableAudioOnly](#meth-disableAudioOnly)
```

Comment:
This should not be needed, as `setVideoTrack` already an almost equivalent job, no?

I said almost because the RxPlayer wouldn't have the memory of the previous seen video track. But I don't know if it's important or not, the application could also well store that information somewhere.

Also, explicitely calling it `AudioOnly` is weird as here subtitles would still be displayed unless also disabled.

---

For that lass point, it all depends on exactly what we want to do.

I'm for copying the behavior than what we have for text tracks (e.g. `disableVideoTrack` and `setTextTrack`) as I see it as the most flexible and elegant (no difference between type of buffers) way to code it.

But I understand if it also makes sense to set APIs explicitely for an audio-only mode instead, as it can speak more to an application developper. APIs like `enableAudioOnly` and `disableAudioOnly`.

What is your opinion?